src/pyfme/aircrafts/cessna_310.py

```diff
@@ -13,21 +13,21 @@
 import numpy as np
 
 from pyfme.utils.anemometry import calculate_dynamic_pressure
+from pyfme.utils.coordinates import hor2body
 
 
-def geometric_Data():
+def geometric_data():
 
     """ Provides the value of some geometric data.
 
     Returns
     ----
-
     Sw : float
-         Wing surface (ft2 * 0.3048 * 0.3048 = m2)
+         Wing surface (m2).
     c : foat
-        Mean aerodynamic Chord (ft * 0.3048 = m)
+        Mean aerodynamic Chord (m).
     span : float
-         Wing span (ft * 0.3048 = m)
+         Wing span (m).

```

Comment:
I would not get too obsessed with periods at the end of comments, I tend to forget them :sweat_smile: